app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+  before_action :authenticate_admin
+
+  private
+
+  def authenticate_admin
+    raise Pundit::NotAuthorizedError unless current_person
```

Comment:
Should we have to manually raise a pundit error, or is this something that pundit should be doing for us? 

...er I guess this is a guard so that we aren't passing `nil` to the policy?